staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter.go

```diff
@@ -42,9 +42,11 @@ const ConfigConsumerAsFieldManager = "api-priority-and-fairness-config-consumer-
 // Interface defines how the API Priority and Fairness filter interacts with the underlying system.
 type Interface interface {
 	// Handle takes care of queuing and dispatching a request
-	// characterized by the given digest.  The given `workEstimator` will be
-	// invoked with the results of request classification and must return the
-	// work parameters for the request.  If the
+	// characterized by the given digest.  The given `noteFn` will be
+	// invoked with the results of request classification.
+	// The given `workEstimator` will be invoked if the
+	// request is not classified as 'exempt' and it must
```

Comment:
The word "it" refers to the most recent noun phrase, which in this case is "the request" --- but that is not what you mean.  Strike the word "it" here.